### ISSUE TITLE
Keen to help? Join as a collaborator!

### ISSUE THREAD

**Issue by twostraws** [twostraws/ControlRoom#182]
Hello! We've had almost 100 PRs merged into Control Room now, and I'm keen to get more people involved. If you've had a pull request merged in already and would like to help more, I'd be happy to add you as a collaborator so you can push code directly, merge PRs from others, and otherwise help provide direction for the project.

If you're interested, reply below with a link to your PR! It doesn't need to be huge – all help is welcome 🙌

**Comment by lechuckcaptain** [comment]
Would be happy to help! (https://github.com/twostraws/ControlRoom/pull/175)

**Comment by ApiOSDev** [comment]
Great to hear! Count me in. Here's my merged PR (#166).  Excited to contribute more! 🎉

**Comment by Asheshp23** [comment]
Great to hear! Count me in. Here's my merged PR (https://github.com/twostraws/ControlRoom/pull/166). Excited to contribute more! 🎉

**Comment by Mcrich23** [comment]
@twostraws I wonder if sandboxing would be difficult considering the requirement to access `xcrun`.

**Comment by twostraws** [comment]
@nickkaczmarek It might simply be the case that if the user manually selects their Xcode version using a File Open dialog, we are allowed to use `xcrun` directly?

**Comment by nickkaczmarek** [comment]
> [@nickkaczmarek](https://github.com/nickkaczmarek) It might simply be the case that if the user manually selects their Xcode version using a File Open dialog, we are allowed to use `xcrun` directly?

@twostraws that's a great point. I always forget about that escape hatch with sandbox folder permissions. I believe you're right that the user enables access to an entire folder so long as they choose it from a File Open Dialog.

Putting this in the mac app store could end up removing the need for something I'm writing up right now RE: automated releases, homebrew, etc. And that is that we first (may) need to automating tagging on successful PR merges so that we can ensure we always have a build with each PR. That may be a bigger discussion though. I can start a new issue once I've finalized my thoughts and some options. I just really want the ability to be able to get the latest build without needing to compile it myself.

**Comment by nickkaczmarek** [comment]
> I would be very happy indeed to have Control Room on the App Store as a free download!

@twostraws pulling on that string some more, do you envision a world in which each merge to main leads to a update tag and push to the app store via CI?

Or would you prefer to handle that ad-hoc?

I'm trying to figure out if I should invest in coming up with some sort of CI process that enables this.

**Comment by twostraws** [comment]
@nickkaczmarek Yes, that sounds more reasonable – if there's a problem, folks can always get the source and build it themselves.
